specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1282,10 +1282,10 @@
         }
       }
     },
-    "AirAndPollen": {
+    "AirQuality": {
       "type": "object",
       "properties": {
-        "name": {
+        "description": {
```

Comment:
You cannot just change the name directly otherwise the deserialization of API response may fail in some languages I would assume. Should use `x-ms-client-name` instead.

Reply:
And there is a property referencing `AirAndPollen` as well.

Reply:
There are other fields under "properties" section where `x-ms-client-name` should be used.

Reply:
yeah I have been trying to fix these check errors. Thanks for reminding :)